node/runtime/pangolin/src/lib.rs

```diff
@@ -940,34 +944,6 @@ pub struct CustomOnRuntimeUpgrade;
 impl OnRuntimeUpgrade for CustomOnRuntimeUpgrade {
 	#[cfg(feature = "try-runtime")]
 	fn pre_upgrade() -> Result<(), &'static str> {
-		// --- paritytech ---
```

Comment:
Would you do some recap to help me understand the upstream migration issues?, it takes me some time to catch up.

Reply:
Find the reason why the key does not equal.
```
[135, 141, 67, 77, 97, 37, 180, 4, 67, 254, 17, 253, 41, 45, 19, 164]    
2022-01-13 00:55:50 pre-migration prefix of storage 'Tips': 'Treasury' ==> 'Tips'    
2022-01-13 00:55:50 pre-migration prefix of storage 'Reasons': 'Treasury' ==> 'Tips'    
2022-01-13 00:55:50 panicked at 'assertion failed: new_pallet_prefix_iter.all(|key| key == storage_version_key)', /Users/denny/.cargo/git/checkouts/substrate-a7fa553ead635512/3d41798/frame/tips/src/migrations/v4.rs:125:5    
Error: Input("failed to execute 'TryRuntime_on_runtime_upgrade': Other(\"Wasm execution trapped: wasm trap: unreachable\\nwasm backtrace:\\n    0: 0xb0ee - <unknown>!rust_begin_unwind\\n\")")
```